docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -401,6 +411,20 @@ Key Interfaces
 
         The EllipticCurvePublicKey object for this private key.
 
+    .. method:: sign(data, signature_algorithm)
+
+        .. versionadded:: 1.5
+
+        Sign one block of data which can be verified later by others using the
+        public key.
+
+        :param bytes data: The message string to sign.
+
+        :param signature_algorithm: An instance of a
+            :class:`EllipticCurveSignatureAlgorithm` provider.
```

Comment:
Yeah, we're trying to drop the "provider" language.

On Sat, Jul 2, 2016 at 11:57 AM, Aviv Palivoda <notifications@github.com>
wrote:

> In docs/hazmat/primitives/asymmetric/ec.rst
> <https://github.com/pyca/cryptography/pull/3029#discussion_r69379578>:
>
> > @@ -401,6 +411,20 @@ Key Interfaces
> >
> >          The EllipticCurvePublicKey object for this private key.
> >
> > +    .. method:: sign(data, signature_algorithm)
> > +
> > +        .. versionadded:: 1.5
> > +
> > +        Sign one block of data which can be verified later by others using the
> > +        public key.
> > +
> > +        :param bytes data: The message string to sign.
> > +
> > +        :param signature_algorithm: An instance of a
> > +            :class:`EllipticCurveSignatureAlgorithm` provider.
>
> This is not consistent with the signer method documentation. Should I
> changed there too?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pyca/cryptography/pull/3029/files/2f973d491640c629ca2f6510dc40175134689486#r69379578>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAADBLJP2Xl9Pb_8-cusavL2SK5IBg2Eks5qRopzgaJpZM4JCkqm>
> .
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6